src/object/object_type.h

```diff
@@ -133,6 +133,7 @@ enum ObjectType
     OBJECT_MOBILEtb         = 221,  //!< TrackedBuilder
     OBJECT_MOBILEfb         = 222,  //!< WingedBuilder
     OBJECT_MOBILEib         = 223,  //!< LeggedBuilder
+    OBJECT_MOBILEpr         = 224,  //!< PracticeBot (alias)
```

Comment:
If you seriously have to add an object type that does nothing and is used just as an alias (and I see no better option of doing it), at least make it stand out so that nobody tries to use it as object type. Maybe add ALIAS to the name and give it some big (or negative even) numeric value?

Reply:
Note that all other aliases use one of the existing values as a 'base' value

Reply:
Seems that nobody knows what do you mean by other aliases and base value. Since this PR is already too old and the issue seems minor, I'm resolving this conversation.